.github/workflows/release.yml

```diff
@@ -21,18 +21,16 @@ jobs:
   meta:
     timeout-minutes: 5
     runs-on: ubuntu-latest
-    container:
-      image: docker://rust:1.60
     steps:
       - id: meta
         shell: bash
         run: |
-          ref=${{ github.ref }}
-          if [[ "$ref" == refs/tags/release/* ]]; then
+          ref='${{ github.ref }}'
+          if [[ "$ref" =~ ^refs/tags/v[0-9]+\.[0-9]+\.[0-9]+ ]]; then
             echo ::set-output name=publish::true
             echo ::set-output name=version::"${ref##refs/tags/release/}"
           else
-            sha=${{ github.sha }}
+            #sha=${{ github.sha }}
```

Comment:
is this supposed to be commented out? should we just remove it (and remove the reference to `sha` on the next line)?

Reply:
good catch. shouldn't be commented